plugins/SpectrumAnalyzer/SaConstants.h

```diff
@@ -0,0 +1,76 @@
+/*
+ * SaConstants.h - Some constants used in spectrum analyser
+ *
+ * Copyright (c) 2025 Roshan M R (Ross Maxx) <mrroshan127/at/gmail/dot/com>
+ *
+ * This file is part of LMMS - https://lmms.io
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public
+ * License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this program (see COPYING); if not, write to the
+ * Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ * Boston, MA 02110-1301 USA.
+ *
+ */
+
+#ifndef SA_CONSTANTS_H
+#define SA_CONSTANTS_H
```

Comment:
This needs the `LMMS_` header guard prefix.

```suggestion
#ifndef LMMS_SA_CONSTANTS_H
#define LMMS_SA_CONSTANTS_H
```